ntoskrnl/mm/section.c

```diff
@@ -4070,9 +4070,6 @@ MmMapViewOfSection(IN PVOID SectionObject,
         IsAttached = TRUE;
     }
 
-    /* FIXME: We should keep this, but it would break code checking equality */
-    Protect &= ~PAGE_NOCACHE;
```

Comment:
It would be nice to mention in the commit log that you're removing a hack that has been introduced in commit https://github.com/reactos/reactos/commit/aa52dc7498dafc07c51919e5210fd0505b4c1930 (r68611) and https://github.com/reactos/reactos/commit/33d53d44da878f83cd117f58ab45c71614905295 (r68612) , for JIRA commit [CORE-9808](https://jira.reactos.org/browse/CORE-9808).